docs/projects/hello-world/_posts/2018-12-05-c.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Hello World in C
+layout: default
+last-modified: 2018-12-05
+featured-image:
```

Comment:
I believe the Hello World in C featured image should be in the images folder. You may be able to get away with putting `hello-world-in-c-featured-image.JPEG` here.